### PR TITLE
Add list method into UsersController

### PR DESCRIPTION
**Pull request by SvetlanaTselikova**
- [x] Add list method

- [x] Add test for list method

- [x] Add isAdmin method

- [x] Add exceptions

https://github.com/malinink/easy-tests/issues/320

### CODE REVIEW

**Comment by SvetlanaTselikova** [comment]
@malinink look,please

**Comment by malinink** [comment]
@SvetlanaTselikova seems to be OK.
Wait for ACL and test implementation in my PR, than you could add access check and tests.

**Comment by malinink** [comment]
@SvetlanaTselikova you could write tests on that, pull dev also.

**Comment by SvetlanaTselikova** [comment]
@malinink look,please

File: src/test/java/easytests/api/v1/controllers/UsersControllerTest.java [hunk 1]
```diff
@@ -1,14 +1,95 @@
...
+@WebMvcTest(controllers = UsersController.class, secure = false)
 public class UsersControllerTest {
+    private static String id = "id";
+    private static String firstName = "firstName";
+    private static String lastName = "lastName";
+    private static String surname = "surname";
+    private static String email = "email";
+    private static String isAdmin = "isAdmin";
+    private static String state = "state";
+    private static String subjects = "subjects";
```
Comment (malinink) [inline src/test/java/easytests/api/v1/controllers/UsersControllerTest.java]: unused variable

**Review by malinink — CHANGES_REQUESTED** [review]

**Comment by malinink** [comment]
@SvetlanaTselikova Why don't you just run tests localy?

**Comment by SvetlanaTselikova** [comment]
@malinink I didn't see checkstyle mistakes localy

**Comment by SvetlanaTselikova** [comment]
@malinink please,look

File: src/test/java/easytests/api/v1/controllers/UsersControllerTest.java [hunk 2]
```diff
@@ -1,14 +1,113 @@
...
+                                .with(state, usersModels.get(1).getState()))
+                        .build()
+                ))
+                .andReturn();
+    }
+
+    @Test
+    public void testListForbidden() throws Exception {
+        final UserModelInterface userModel = new UserModel();
+        userModel.map(this.usersSupport.getEntityFixtureMock(1));
```
Comment (malinink) [inline src/test/java/easytests/api/v1/controllers/UsersControllerTest.java]: It is hard to understand that user with idx0 is admin and with idx1 is not.
I think that a good idea is:
manually set is admin to be true or false for each case.
Than, if we modify support class data test will not fail.

But there is one more clean case:
 - add two methods into users support class: getAdminUser and getNoneAdmoinUser and act with that users or smt like that.

**Review by malinink — CHANGES_REQUESTED** [review]

**Comment by SvetlanaTselikova** [comment]
@malinink look again,please

**Review by malinink — APPROVED** [review]

**Pull request merged**